### ISSUE TITLE
Issue with the code 

### ISSUE THREAD

**Issue by SHIKA99** [sauriii98/Motor-Imagery-Classification#1]
Hello,
I'm currently trying to run this code, but I'm having issues with this part of the code and the error states as below: 

---------------------------------------------------------------------------
KeyError                                  Traceback (most recent call last)
[<ipython-input-54-cacbbbbd9fb0>](https://localhost:8080/#) in <cell line: 2>()
      1 idx = 4
----> 2 marker = get_markers(mat, idx)

[<ipython-input-49-ee04fb31bdb6>](https://localhost:8080/#) in get_markers(mat_data, index)
      5 
      6     # Example code:
----> 7     markers = mat_data['markers'][index]
      8     return markers
      9 idx = 4

KeyError: 'markers'

I have attached the part where the error is found. Hope you could help me out!!
![Screenshot 2023-06-16 022105](https://github.com/sauriii98/Motor-Imagery-Classification/assets/135220845/167732a2-7b2a-4e7b-a27a-e0cd7e4ec514)

Thankk youu!!

**Comment by fina1e** [comment]
Yes, it needs to be fixed. Just delete the idx for this function.
